src/Components/Blueprints/ImportBlueprintModal.tsx

```diff
@@ -49,17 +60,28 @@ export const ImportBlueprintModal: React.FunctionComponent<
     setJsonContent(value);
   };
   const handleDataChange = (_: DropEvent, value: string) => {
-    setJsonContent(value);
+    try {
+      const importedBlueprint: BlueprintResponse = JSON.parse(value);
+      const importBlueprintState = mapRequestToState(importedBlueprint);
+      setImportedBlueprint(importBlueprintState);
+      setJsonContent(value);
+    } catch (error) {
+      console.log(error);
```

Comment:
I don't think we should be logging errors to the console as those have very little value to user.

Since this would be an error with the blueprint format, would a toast notification like "Failed to parse the blueprint" or something similar work?

Reply:
Oh yeah, you are right. This was a leftover from testing. :eyes: